doc/source/spectra.rst

```diff
@@ -317,6 +317,74 @@ Plotting
 
 See :ref:`Plotting <plotting>`
 
+Kinematic constraints
+---------------------
+
+Inelastic neutron-scattering (INS) experiments are often performed on
+time-of-flight (ToF) spectrometers where a wide ToF range yields a wide
+range of energy transfers which are measured simultaneously.
+In "direct geometry" the incident energy is fixed (e.g. by a Fermi
+chopper) while in "indirect geometry" the scattered energy is fixed
+(e.g. by scattering from an "analyser" crystal).
+Conservation laws allow the overall energy and momentum
+transfer to be determined for a given scattering angle and crystal
+orientation. In powder measurements, the crystal orientation is not
+needed so the kinematic limits --- the accessible :math:`(q, \omega)`
+range --- determined by the conservation laws are given solely
+by these instrument parameters.
+
+The function :py:func:`apply_kinematic_constraints
+<euphonic.spectra.apply_kinematic_constraints>` applies these limits
```

Comment:
It would be good to show the function is `euphonic.spectra.apply_kinematic_constraints` otherwise it's not clear where to import it from!